guides/common/attributes-base.adoc

```diff
@@ -13,21 +13,9 @@
 :UpgradingDocURL: {BaseURL}Upgrading_and_Updating/{BaseFilenameURL}#
 :ReleaseNotesURL: {BaseURL}Release_notes/{BaseFilenameURL}#
 
+
 // Repositories and subscriptions (used both in Katello and Satellite guides)
-:SatelliteSub: Red Hat Satellite Infrastructure Subscription
-:RepoRHEL7Server: rhel-7-server-rpms
-:RepoRHEL7ServerSoftwareCollections: rhel-server-rhscl-7-rpms
-:RepoRHEL7ServerOptional: rhel-7-server-optional-rpms
-:RepoRHEL7ServerAnsible: rhel-7-server-ansible-2.9-rpms
```

Comment:
It was suggested to keep these in here since these are also used for upstream installations on RHEL 7 in all scenarios, so base is really the correct place.

Reply:
Ok, got it. So I've moved these to attributes-base.adoc a removed from attributes-satellite.adoc. Also removed all I've added into attributes-katello.adoc. So hope it's correct now :-) Thank you for your guidance!